db/migrate/20150502104737_add_member_to_user.rb

```diff
@@ -0,0 +1,13 @@
+class AddMemberToUser < ActiveRecord::Migration
+  def up
+    add_column :users, :member_at, :datetime
+    # To avoid a short time window between running the migration and updating all existing
+    # users as members, do the following
+    execute("UPDATE users SET member_at = NOW()")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.